registry/storage/manifeststore.go

```diff
@@ -1,163 +0,0 @@
-package storage
-
-import (
-	"fmt"
-
-	"github.com/docker/distribution"
-	"github.com/docker/distribution/context"
-	"github.com/docker/distribution/digest"
-	"github.com/docker/distribution/manifest/schema1"
-	"github.com/docker/distribution/reference"
-	"github.com/docker/libtrust"
-)
-
-type manifestStore struct {
```

Comment:
@stevvooe: My followup PR will rename this file and do some refactoring to support both schema versions.